src/main/java/co/inventorsoft/academy/spring_boot_task/config/TournamentConfig.java

```diff
@@ -0,0 +1,9 @@
+package co.inventorsoft.academy.spring_boot_task.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = "co.inventorsoft.academy.spring_boot_task")
```

Comment:
This is redundant since @SpringBootApplication annotation will add automatic scanning for base package